src/main/kotlin/controller/Controller.kt

```diff
@@ -11,38 +12,71 @@ class Controller {
         inputView: InputView,
         outputView: OutputView,
     ) {
-        val lotto = handleLottoPurchase(inputView, outputView)
+        val lotto = setNumbersOfEachTickets(inputView, outputView)
+        fillNumbersToEachTickets(lotto, inputView, outputView)
+        outputView.displayNumberOfLottoTickets(lotto)
+        outputView.displayTickets(lotto)
```

Comment:
I see there's an `outputview` used twice in a row.
Why don't we make it so that one method handles it all?
When we need to separate these displays into two for logical or graphical reason, that's when dividing them could be great.